src/parser/test/ParserTest.cpp

```diff
@@ -1323,18 +1323,6 @@ TEST(Parser, ConfigOperation) {
         auto result = parser.parse(query);
         ASSERT_TRUE(result.ok()) << result.status();
     }
-    {
-        GQLParser parser;
-        std::string query = "UPDATE CONFIGS load_data_interval_secs=120";
```

Comment:
We'd better keep this test. It is still used in console. We could change `load_data_interval_secs` to `v`.

Reply:
OK!